packages/admin-ui/src/Radio/RadioGroupPresenter.ts

```diff
@@ -0,0 +1,48 @@
+import { makeAutoObservable } from "mobx";
+import { RadioGroupVm } from "./RadioGroup";
+import { RadioItem } from "~/Radio/RadioItem";
+import { RadioItemMapper } from "~/Radio/RadioItemMapper";
+import { RadioItemDto } from "~/Radio/RadioItemDto";
+
+interface RadioGroupPresenterParams<TValue = string> {
+    items: RadioItemDto[];
```

Comment:
Still on the fence whether this is really `RadioItemDto`. :D 

Maybe `RadioItemParams`? 

IDK, I guess LR video effed my brain and hard-wired the DTO term to gatway layer 😵 .

Reply:
DTOs encapsulate and transfer data between application layers or components, so they are not wired to the gateway layer only.

Anyway, I changed `RadioItemDto` into `RadioItemParams` as you suggest. It makes sense 👍